content/root/platform-customization.textile

```diff
@@ -0,0 +1,147 @@
+
+If you are not currently an enterprise customer but are interested in finding out more, then please "get in touch":https://ably.com/contact.
+
+If you're ready to enable a custom environment to take advantage of these features, then please "see our guide":#custom-environment-setup. Otherwise, read on to find out more.
+
+
+h2(#active-traffic-management). Active traffic management
+
+Active traffic management (ATM) is a feature available exclusively to our Enterprise customers that enables the Ably engineering team to automatically and manually manage the endpoints that customers connect to.
+
+This ensures that under adverse conditions, such as a denial of service attack or a major outage, Ably can proactively route Enterprise customer traffic to specific data centers or regions that are unaffected, or even to new temporary clusters. This enables customers to maintain service while the issue is investigated and resolved.
+
+This service is ideal for customers who want the piece of mind that Ably actively monitors not only the global cluster health but also the endpoints used by that customer, and can act to isolate or move traffic, ensuring business continuity.
+
+h2(#dedicated-and-isolated-clusters). Dedicated, isolated clusters
+
+As part of our Enterprise package offering, we are able to offer customers dedicated clusters that are either partially or completely isolated from Ably's global cluster.  Customers who require isolation from our global cluster for security, governance or guaranteed capacity reasons are able to run the Ably platform on their own EC2 dedicated environments.  
+
+This involves the provision of dedicated realtime and routing servers, with shared database cluster and common services (such as stats, logging, registries, and configuration).
+
+We provide you with custom endpoints for your cluster, for example: @yourcompany-realtime.ably.io@ and @yourcompany-realtime.ably.io@.  Additionally, we can support CNAMEs using your own custom root domain.
+
+Please note the following:
+- Ably only supports Amazon EC2 environments (within Virtual Private Networks)
+- Ably is responsible for pro-actively managing and updating dedicated clusters
+- Dedicated clusters can run in one or more regions of your choice
+ 
+If you would like to find out more about our dedicated cluster offering, please "get in touch":https://ably.com/contact.
+
+h2(#regional-routing-and-storage). Regional routing and storage
+
+If you need your data to remain within either Europe or the US, we can customize your account so that all of your data (both stored data and transient realtime data transmission activity) is kept in servers within the EU or US.
```

Comment:
```suggestion
If you need your data to remain within either Europe or the US, Ably can customize your account so that all of your data (both stored data and transient realtime data transmission activity) is kept in servers within the EU or US.
```